config/routes.rb

```diff
@@ -10,7 +10,11 @@
   #ENC requests goes here
   get "node/:name" => 'hosts#externalNodes', :constraints => { :name => /[^\.][\w\.-]+/ }
 
-  resources :reports, :only => [:index, :show, :destroy] do
+  # Redirect reports with deprecation warning
+  get "/reports", :to => redirect { |params, request| DeprecationRedirect.new("config_reports", params, request.query_string, '1.12', 'Reports are renamed to Config Reports').redirect_url }
```

Comment:
It is supported in Rails 3.2 also.
Added initially cause I thought that "/reports" may be bookmarked for some users.
removed